app/controllers/upload_controller.rb

```diff
@@ -0,0 +1,6 @@
+class UploadController < ApplicationController
```

Comment:
you can just delete the view and controller files. If you used the rails generate function you can delete it like this: http://stackoverflow.com/questions/4161357/how-do-i-reverse-a-rails-generate